netdata - new version 1.3.0/changes - program error build/compile - @diizzyy

@diizzyy
make -j8 BUILD_LOG=1

cat [sources]/logs/package/feeds/packages/netdata/compile.txt:

```
make[4]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0'
make  all-recursive
make[5]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0'
Making all in charts.d
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/charts.d'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/charts.d'
Making all in conf.d
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/conf.d'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/conf.d'
Making all in node.d
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/node.d'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/node.d'
Making all in plugins.d
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/plugins.d'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/plugins.d'
Making all in python.d
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/python.d'
make[6]: Nothing to be done for 'all'.
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/python.d'
Making all in src
make[6]: Entering directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/src'
powerpc-openwrt-linux-musl-gcc      -Os -pipe -mcpu=464fp -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -iremap /home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0:netdata-1.3.0 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro    -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/lib -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/lib -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/toolchain-powerpc_464fp_gcc-5.4.0_musl-1.1.15/usr/lib -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/toolchain-powerpc_464fp_gcc-5.4.0_musl-1.1.15/lib -znow -zrelro  -o netdata appconfig.o avl.o common.o daemon.o dictionary.o eval.o global_statistics.o health.o log.o main.o plugin_checks.o plugin_idlejitter.o plugin_nfacct.o plugin_proc.o plugin_tc.o plugins_d.o popen.o proc_diskstats.o proc_interrupts.o proc_softirqs.o proc_loadavg.o proc_meminfo.o proc_net_dev.o proc_net_ip_vs_stats.o proc_net_netstat.o proc_net_rpc_nfsd.o proc_net_snmp.o proc_net_snmp6.o proc_net_stat_conntrack.o proc_net_stat_synproxy.o proc_stat.o proc_self_mountinfo.o proc_sys_kernel_random_entropy_avail.o proc_vmstat.o sys_kernel_mm_ksm.o sys_fs_cgroup.o procfile.o registry.o rrd.o rrd2json.o storage_number.o unit_test.o url.o web_buffer.o web_buffer_svg.o web_client.o web_server.o -lm  -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/lib -lz -L/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/lib -luuid  
global_statistics.o: In function `finished_web_request_statistics':
global_statistics.c:(.text+0x9c): undefined reference to `__atomic_compare_exchange_8'
global_statistics.c:(.text+0xc8): undefined reference to `__atomic_fetch_add_8'
global_statistics.c:(.text+0xdc): undefined reference to `__atomic_fetch_add_8'
global_statistics.c:(.text+0xf0): undefined reference to `__atomic_fetch_add_8'
global_statistics.c:(.text+0x104): undefined reference to `__atomic_fetch_add_8'
global_statistics.c:(.text+0x118): undefined reference to `__atomic_fetch_add_8'
global_statistics.o:global_statistics.c:(.text+0x12c): more undefined references to `__atomic_fetch_add_8' follow
global_statistics.o: In function `global_statistics_copy':
global_statistics.c:(.text+0x334): undefined reference to `__atomic_compare_exchange_8'
collect2: error: ld returned 1 exit status
Makefile:575: recipe for target 'netdata' failed
make[6]: *** [netdata] Error 1
make[6]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/src'
Makefile:461: recipe for target 'all-recursive' failed
make[5]: *** [all-recursive] Error 1
make[5]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0'
Makefile:400: recipe for target 'all' failed
make[4]: *** [all] Error 2
make[4]: Leaving directory '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0'
Makefile:60: recipe for target '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/.built' failed
make[3]: *** [/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/netdata-1.3.0/.built] Error 2
```

--- a/admin/netdata/Makefile
+++ b/admin/netdata/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netdata
 PKG_VERSION:=1.3.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 PKG_MAINTAINER:=Sebastian Careba <nitroshift@yahoo.com>
 PKG_LICENSE:=GPL-3.0
 PKG_LICENSE_FILES:=COPYING
@@ -21,6 +21,8 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 
 PKG_INSTALL:=1
 PKG_FIXUP:=autoreconf
+
+TARGET_CFLAGS+= -DNETDATA_NO_ATOMIC_INSTRUCTIONS=1
 
 include $(INCLUDE_DIR)/package.mk
 

